### ISSUE TITLE
mjpeg_vaapi

### ISSUE THREAD

**Issue by randellhodges** [pvizeli/ha-ffmpeg#21]
https://github.com/pvizeli/ha-ffmpeg/blob/898491671e8dc60b140a0353b0cddeb234e11906/haffmpeg/camera.py#L15

I'm attempting to pass in via the extra command line arguments in HA the parameters I need to use mjpeg_vaapi. One of the arguments I'm passing in is 
`-c:v mjpeg_vaapi`

When I debug, I see 
`'-c:v', 'mjpeg', '-vaapi_device', '/dev/dri/renderD128', '-c:v', 'mjpeg_vaapi'`

I'm not sure if ffmjpeg will ignore the first, or ignore the second.

Would it be possible only default the -c:v if one isn't passed in?

**Comment by pvizeli** [comment]
Yeah, the only filter would be merged. I think that need as an option into mjpeg and Home Assistant
